static/src/javascripts/projects/commercial/modules/prebid/bidder-config.js

```diff
@@ -57,30 +59,118 @@ const getIndexSiteId = (): string => {
     return site ? site.id : '';
 };
 
+const contains = (sizes: PrebidSize[], size: PrebidSize): boolean =>
+    Boolean(sizes.find(s => s[0] === size[0] && s[1] === size[1]));
+
+const getImprovePlacementId = (sizes: PrebidSize[]): number => {
+    switch (config.page.edition) {
+        case 'UK':
+            switch (getBreakpointKey()) {
+                case 'D':
+                    if (
+                        contains(sizes, [300, 250]) ||
```

Comment:
you might want to add a `const` for this boolean logic, and another for the banner sizes.